Wireframe/index.html

```diff
@@ -3,30 +3,67 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
+    <title>SHIRAZ</title>
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>SHIRAZ</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        This is the picture of Shiraz my beautiful city.
       </p>
+
     </header>
+
     <main>
+      
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="shiraz3.jpg" alt="" />
```

Comment:
It looks like you have three image tags in this file, but I don't see any of the images in the rendered deploy preview at https://deploy-preview-216--cyf-onboarding-module.netlify.app/wireframe/ - do you know why they're not showing up? Can you make them show up?

Also you're not setting any alt text for any of them at the moment - can you explain what alt text is for? Why is it important? Did you choose to leave it empty here? If so, why?